docs/test-suite/ReFrame-configuration-file.md

```diff
@@ -434,3 +436,182 @@ reframe --list
 ```
 
 ReFrame will store the processor information for your system in `~/.reframe/topology/<system>-<partition>/processor.json`.
+
+#### Create topology file
+
+You can also use the reframe option `--detect-host-topology` to create the topology file yourself.
+
+Run the followiging command on the cluster of which you need the topology.
+
+```bash
+reframe --detect-host-topology[=FILE]
+```
+
+The output will be put in a file if this is specified or printed in the output. It will look something like this.
+
+```json
```

Comment:
Since this quite long, maybe we should put in in a "details" box that can be expanded?

````
<details>

```json
{
  "arch": "skylake_avx512",
...
```

</details>
````

(note that you need a blank line below `<details>` and above `</details>` when putting a code block in it